SETUP.md

```diff
@@ -0,0 +1,78 @@
+# Contributing
+
+Thank you for considering contributing to the R Markdown C
+
+These instructions describe the steps and system requirements needed to build this book. 
+
+When contribution to the book, it is a good practice to build it locally to ensure that any modifications that you have made worked as expected. However, please do not include the built HTML or PDF in your pull request. It is easier to merge in plain text R, Markdown, and R Markdown files and to only reknit the book on the master branch. 
+
+## System Requirements
+
+### R Packages
+
+The following R packages are required to build to book. These are the same as those which are loaded in the `index.Rmd` file, plus `bookdown` and `tinytex`.
+
+```
+reqd_pkgs <-
+c('remotes', 'webshot', 'DiagrammeR', 'stargazer', 'gt', 'flextable', 'formatR',
+  'equatiomatic', 'gifski', 'magick', 'pdftools', 'tikzDevice', 'xaringan',
+  'kableExtra', 'blogdown', 'jsonlite', 'nomnoml', 'bookdown', 'tinytex')
+```
```

Comment:
I was thinking that even though they are installed, it might be nice to just install them explicitly. Otherwise, the first knit could take a very long time and be disconcerting. That's just a weird personal preference, though, so I can remove. The main downside to keeping it is maintaining them both if we add new packages. What do you think?

Reply:
> The main downside to keeping it is maintaining them both if we add new packages. 

Yeah that is what I found not so great. Maybe we should switch to one way to install it in that case: DESCRIPTION or renv.
I think yihui said it is our book for Second Edition, we can choose ! 😄